contractcourt/chain_arbitrator.go

```diff
@@ -514,13 +514,16 @@ func (c *ChainArbitrator) Stop() error {
 // NOTE: This must be launched as a goroutine.
 func (c *ChainArbitrator) watchForChannelClose(closeInfo *channeldb.ChannelCloseSummary) {
 	spendNtfn, err := c.cfg.Notifier.RegisterSpendNtfn(
-		&closeInfo.ChanPoint, closeInfo.CloseHeight, true,
+		&closeInfo.ChanPoint, closeInfo.CloseHeight, false,
```

Comment:
Similar comment here w.r.t also enabling callers to wait on an additional set of confirmations.